libs/components/lists/src/lib/modules/filter/filter-button.component.scss

```diff
@@ -20,16 +20,64 @@
   .sky-filter-btn-active,
   .sky-filter-btn-active:hover,
   .sky-filter-btn-active:focus {
-    color: initial;
+    background-color: var(--sky-color-background-selected-soft);
```

Comment:
This file still has the import for 'libs/components/theme/src/lib/styles/_public-api/themes/modern/_compat/mixins'. I believe that can be removed now.